app/uk/gov/hmrc/traderservices/models/DeclarationDetails.scala

```diff
@@ -22,7 +22,9 @@ import play.api.libs.json.{Format, Json}
 
 case class DeclarationDetails(epu: EPU, entryNumber: EntryNumber, entryDate: LocalDate) {
 
-  val isExportDeclaration: Boolean = entryNumber.value.headOption.forall(_.isLetter)
+  val isExportDeclaration: Boolean =
+    entryNumber.value.headOption.forall(_.isLetter) && entryNumber.value.lastOption.forall(_.isLetter)
+  val isImportDeclaration: Boolean = entryNumber.value.lastOption.forall(_.isLetter)
```

Comment:
This will give true also for a valid export number, ending in a letter too.

Reply:
@rhague-hmrc This one too, please. Thanks!